README.md

```diff
@@ -5,26 +5,25 @@ Back-tick template engine for JavaScript
 ## Usage
 
 - Install: `npm install tickplate`
-- Require: `const t = require('tickplate');`
-- Place tag `t` before templated string
+- Require: `const tpl = require('tickplate');`
+- Call `tpl` function like that: `tpl(filename, data, your_callback(err, data))`
 
 ## Examples:
 
-```js
-const t = require('tickplate');
+```javascript
+const tpl = require('./tickplate.js');
```

Comment:
This change is kinda incorrect. Examples show how to use the library, not to hack with its sources, and anyone using the package will certainly install it with a package manager and do `require('tickplate')`.